packages/appstore/src/components/wallet-header/wallet-header.scss

```diff
@@ -17,12 +17,19 @@
         width: 100%;
         height: 100%;
         opacity: 0.1;
-        background-image: url('./../../public/images/wallet-demo-bg.svg');
         background-repeat: repeat;
         background-position: 3% 30%;
         border-radius: $BORDER_RADIUS * 4;
     }
 
+    .theme--light &__demo:before {
+        background-image: url('./../../public/images/wallet-header-demo-bg.svg');
+    }
+
+    .theme--dark &__demo:before {
+        background-image: url('./../../public/images/wallet-header-demo-bg-dark.svg');
+    }
```

Comment:
This is an icon can we use it inside the component we created?

Reply:
@aizad-deriv I can't do this because I'm using this image as background and can't use `Icon` component. It's the same problem as I had earlier when Thisyahlen helped me with webpack config. But for dark theme I have to use another background image and I needed to add this to `public/images` folder too.